models/story.model.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This file is used for connecting MongoDB and query data.
+ */
+
+// Set up mongoose connection
+const mongoose = require('mongoose'),Admin=mongoose.mongo.Admin;

```

Comment:
Was just looking into stuff. Database connection in a model file sounded a bit weird to me.
Warning: not a mongo expert.

Reply:
Yup you are right. We discussed this offline. The code structure is going to change slightly in the next commit. I will get it out as soon as possible